src/apps/SafeConnect/components/WCClientInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+import { Text, IconText, Icon } from "@gnosis.pm/safe-react-components";
+
+const FlexAlignedCenter = styled.div`
+  display: flex;
+  margin-bottom: 10px;
+
+  > :first-of-type {
+    margin-right: 5px;
+  }
+`;
+
+const Image = styled.div<{ src: string }>`
```

Comment:
the name is misleading, this is not an image

any reason why it couldn't be a regular image?

Reply:
1. It's a div simulating to be an image, so the name it's fine. 
2. https://stackoverflow.com/a/1469139/2524608

Reply:
Well that's debatable, IMO our usage falls into this category
```
Pragmatic uses of IMG
Use IMG plus alt attribute if the image is part of the content such as a logo or diagram or person (real person, not stock photo people). —sanchothefat
```

But since we still display the name so in our case using background image is fine

Reply:
the main problem is that we don't know the image size, so using `background` with `cover`, we can ensure the image will look properly.